Resources/views/pjax_admin.html.twig

```diff
@@ -25,3 +25,18 @@
     </header>
     {% block content %}{% endblock %}
 </div>
+
+{% block notification %}
+<div data-name="notification" data-state="{{ notification_state|default("done") }}">
```

Comment:
Can be a `ul` yes. And yes, potentially there might be different messages in different states, even though it's not currently discriminated.

As for `notification_state`, it can be overriden in a child block. If not defined, it defaults to `done`. That's no magic at all :wink: